main/kotlin/com/example/people_managment_system/dtos/EmployeeDto.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.people_managment_system.dtos
+
+import java.io.Serializable
+
+data class EmployeeDto(
+    var employeeID: Int? = null,
+    var employeeName: String? = null,
+    var email: String? = null,
+    var salary: Int? = null,
+    var companyName: String? = null,
+    var managerID: Int? = null,

```

Comment:
I recommend to always follow `camelCase` naming convention for variables in Kotlin.